cli/azd/pkg/project/container_helper.go

```diff
@@ -104,6 +104,18 @@ func (ch *ContainerHelper) Login(
 	return loginServer, ch.containerRegistryService.Login(ctx, targetResource.SubscriptionId(), loginServer)
 }
 
+func (ch *ContainerHelper) Credentials(
+	ctx context.Context,
+	targetResource *environment.TargetResource,
+) (*azcli.DockerCredentials, error) {
+	loginServer, err := ch.RegistryName(ctx)
```

Comment:
Did we ever intend to support non-Azure registries? I realize now that seemed to be an implicit decision, but the code doesn't really support it today. May just be a follow-up issue.

Reply:
It makes sense long term to support non-Azure registries but ACR is our primary registry at the moment.

Reply:
I think the next thing we will want to support is GitHub Container Registries.  I think we have an issue in the backlog about that.

Agree will have to revisit this when we pull that chunk of work off the backlog.